adafruit_bme680.py

```diff
@@ -598,7 +602,7 @@ def _calc_gas_wait(self, dur: int) -> int:
         factor: int = 0
         durval: int = 0xFF  # Max duration
 
-        if dur < 0xFC0:
+        if dur > 0xFC0:
```

Comment:
@CTho9305 the code proposed in #67 is:
```
if dur >= 0xFC0:
```
But in this change `>` is used instead of `>=`

Honestly this change is a bit over my head, I'm not sure what it's doing or how to properly read or interpret the datasheet. Is the difference in greather than vs. greater equal handled by some other portion of this change or is there some other reason to drop the equals portion of this condition?

Reply:
You're right; I intended to match their reference but didn't.  Thanks for double-checking.

This block of code is taking a desired duration and figuring out how to encode it in the form the sensor wants (described in section 5.3.3.3 of the datasheet); up to 63ms is represented directly, but above 63ms the upper bits indicate powers of four to multiply value in the lower bits (I guess a little bit like floating point representations with an exponent and mantissa, to allow for a large range of values in a single byte).

The lower bits can only represent values up to 63, so the code is figuring out what power of four has a remainder of less than or equal to 63 (0x3F).  The largest exponent is 3 (the variable `factor` in this code) representing 64, and the largest representable time is 64*63 = 4032, so this first check is just capping the duration at 4032.

The while loop then does progressive divisions by 4 and finding the point where the remainder becomes representable in 6 bits.

That said, I don't think there's any actual consequence of my mistake, because 0xFC0 is 4032, which the while loop correctly handles, but I still think the code should match the vendor's recommendation so I'm pushing a correction.